auto_round/inference/backend.py

```diff
@@ -121,6 +124,39 @@ def fp8_static_scheme_checker(
     return config == FP8_STATIC
 
 
+SCHEME_CHECK_ATTRS = ["bits", "group_size", "sym", "data_type", "act_bits", "act_group_size", "act_sym"]
+
+
+def _scheme_checker_common(config1: QuantizationScheme, config2: QuantizationScheme):
+    for attr in SCHEME_CHECK_ATTRS:
```

Comment:
how about moving this list inside if only one function uses it. And better change it to tuple